applications/luci-app-irqbalance/htdocs/luci-static/resources/view/irqbalance.js

```diff
@@ -0,0 +1,116 @@
+    },
+
+    render(data) {
+        const cpuNum = data.match(/\bCPU\d+\b/g).map(i => i.slice(3)), // cpuNum = data.match(/(?<=\bCPU)\d+\b/g), // Safari did not support RegExp lookbehind assertion before version 16.4.
+            irqNum = data.match(/\b\d+(?=: )/g);
+        let m, s, o;
+
+        m = new form.Map('irqbalance', _('irqbalance'), _('The purpose of irqbalance is to distribute hardware interrupts across processors/cores on a multiprocessor/multicore system in order to increase performance.'));
+
+        s = m.section(form.NamedSection);
+        s.anonymous = true;
+        s.render = function () {
+            L.Poll.add(function () {
+                return L.resolveDefault(getServiceStatus()).then(function (res) {
+                    let view = document.getElementById('status');
+                    view.innerHTML = renderStatus(res);
+                });
+            });
+
+            return E('div', { class: 'cbi-section' }, [
+                E('p', { id: 'status' }, _('Loading...'))
+            ]);
+        }
+
+        s = m.section(form.TypedSection, 'irqbalance', _('IRQs snapshot'));
+        s.anonymous = true;
+
+        s = m.section(form.NamedSection);
+        s.anonymous = true;
+        s.render = function () {
+            const snapshot = new ui.Textarea(data.slice(0, -1), {
+                readonly: true,
+                placeholder: _('Loading...'),
+                monospace: true,
+                rows: data.split('\n').length - 1,
+            });
+            return snapshot.render();
+        }
+
+        s = m.section(form.TypedSection, 'irqbalance', _('General settings'));
+        s.anonymous = true;
+
+        o = s.option(form.Flag, 'enabled', _('Enable'));
+        o.default = '0';
+        o.rmempty = false;
+
+        o = s.option(form.Value, 'deepestcache', _('Deepest cache'), _('Specify cache level at which irqbalance partitions cache domains.'));
+        o.placeholder = '2';
+        o.datatype = 'uinteger';
+        o.optional = true;
+
+        o = s.option(form.Value, 'interval', _('Interval'), _('Value in seconds.'));
+        o.placeholder = '10';
+        o.datatype = 'uinteger';
+        o.optional = true;
+
+        o = s.option(form.Value, 'banned_cpulist', _('Excluded CPU list'), _('Specify excluded CPU list, can be an integer or integers separated by commas.') + '<br />' + _('Allowed value(s): %s.').format(cpuNum.join(', ')));
+        o.placeholder = '0';
+        o.optional = true;
+        o.validate = function (section_id, value) {
+            for (const i of value.split(',')) {
+                if (!cpuNum.includes(i) && i != '') {
+                    return _('Invalid');
+                }
+            }
+            return true;
+        }
+
+        o = s.option(form.DynamicList, 'banirq', _('Banned IRQs'), _('Specify banned IRQs.') + '<br />' + _('Allowed values: %s.').format(irqNum.join(', ')));
```

Comment:
Here is a setting called "banned" (past tense), and yet in the next row, is "allowed" (past tense). Which is it? Avoid dissonance. 

Does "allowed" here mean "values to choose from"? And "banned" means "to block"?

I note the setting is 'banirq', which is not past tense, but *present* simple tense.